addons/switchunits/functions/fnc_switchBack.sqf

```diff
@@ -10,20 +10,19 @@
  * None
  *
  * Example:
- * [_originalPlayerUnit, _currentUnit] call FUNC(switchBack)
+ * [_originalPlayerUnit, _currentUnit] call ace_switchunits_fnc_switchBack
  *
  * Public: Yes
  */
-
 #include "script_component.hpp"
 
-PARAMS_1(_originalPlayerUnit);
+params ["_originalPlayerUnit"];
 
 [_originalPlayerUnit] joinSilent GVAR(OriginalGroup);
 
-DFUNC(pfhSwitchBack) = {
-    PARAMS_2(_args,_pfID);
-    EXPLODE_2_PVT(_args,_originalPlayerUnit,_currentUnit);
+_fnc_pfhSwitchBack = {
```

Comment:
Missing private. Might be better to just put the code straight into the PFH without the assignment though